packages/sdk/test/bench/splay_tree.bench.ts

```
@@ -0,0 +1,96 @@
+import { describe, bench } from 'vitest';
+import { SplayNode, SplayTree } from '@yorkie-js-sdk/src/util/splay_tree';
+import editTraceData from './editing-trace.json';
+
+class StringNode extends SplayNode<string> {
+  public removed: Boolean = false;
+  constructor(value: string) {
+    super(value);
+  }
```

Biome

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)